docs/source/guide/tutorials/base/how_to_train/action_classification.rst

```diff
@@ -0,0 +1,275 @@
+    │   │   │   │   │   ├── flow_y_00001.jpg
+    │   │   │   │   │   ├── flow_y_00002.jpg
+    │   │   │   ├── ...
+    │   │   │   ├── wave
+    │   │   │   │   ├── 20060723sfjffbartsinger_wave_f_cm_np1_ba_med_0
+    │   │   │   │   ├── ...
+    │   │   │   │   ├── winKen_wave_u_cm_np1_ri_bad_1
+    |
+
+Once you have the dataset structured properly, you can convert it to the `CVAT <https://www.cvat.ai/>`_ format using the following command:
+
+.. code-block::
+
+  (otx) ...$ python3 otx/algorithms/action/utils/convert_public_data_to_cvat.py \
+  --task action_classification \
+  --src_path ./data/hmdb51/rawframes \
+  --dst_path ./data/hmdb51/CVAT/train \
+  --ann_file ./data/hmdb51/hmdb51_train_split_1_rawframes.txt \
+  --label_map ./data/hmdb51/label_map.txt
+
+The resulting folder structure will be as follows:
+
+.. code-block::
+
+    hmdb51
+    ├── rawframes
+    ├── videos
+    ├── annotations
+    └── CVAT
+        ├── train (3570 videos)
+        │    ├── Video_0
+        │    │   ├── annotations.xml
+        │    │   └── images [101 frames]
+        │    ├── Video_1
+        │    │   ├── annotations.xml
+        │    │   └── images [105 frames]
+        │    └── Video_2
+        │        ├── annotations.xml
+        │        └── images [64 frames]
+        │
+        └── valid (1530 videos)
+            ├── Video_0
+            │   ├── annotations.xml
+            │   └── images [85 frames]
+            ├── Video_1
+            │   ├── annotations.xml
+            │   └── images [89 frames]
+            └── Video_2
+                ├── annotations.xml
+                └── images [60 frames]
+
+*********
+Training
+*********
+
+1. Choose which action classification model to train by running the following command to see the list of supported templates:
+
+.. note::
+
+  OTX is supporting only X3D model template now, other architecture will be supported in near future.
+
+.. code-block::
+
+  (otx) ...$ otx find --task action_classification
+
+  +-----------------------+----------------------------------+------+----------------------------------------------------------------+
+  |          TASK         |                ID                | NAME |                           BASE PATH                            |
+  +-----------------------+----------------------------------+------+----------------------------------------------------------------+
+  | ACTION_CLASSIFICATION | Custom_Action_Classificaiton_X3D | X3D  | otx/algorithms/action/configs/classification/x3d/template.yaml |
+  +-----------------------+----------------------------------+------+----------------------------------------------------------------+
+
+All commands will be run on the X3D model. It's a light model, that achieves competitive accuracy while keeping the inference fast.
+
+2. Prepare an OTX workspace for the action classification task by running the following command:
+
+.. code-block::
+
+  (otx) ...$ otx build --task action_classification
```

Comment:
Personally I prefer build with --train-data-roots and --val-data-roots.

Reply:
Added, thank you!